src/analytics/index.ts

```diff
@@ -1,9 +1,17 @@
 import { Identify, identify, init, track } from '@amplitude/analytics-browser'
-import { isProductionEnv } from '../utils/env'
 
 import { ApplicationTransport, OriginApplication } from './ApplicationTransport'
 
+export type Options = {
```

Comment:
naming

1. Why `Options` instead of `Config`? Env / commit has aren't really options IMO
1. This is very general, maybe `AnalyticsConfig`